src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -480,6 +505,103 @@ public override async Task RunAsync()
             await runTask;
         }
 
+        private async Task<bool> HandleRuntimeParam(bool isCurrentProcessNet6Build, bool isVerbose)
```

Comment:
Given this returns a bool, "handle" isn't the right keyword to use here. Ask yourself, what is this method saying true/false to? And choose a name that best describes the purpose and behavior of the method.

Does perhaps "IsRuntimeParamValid" make more sense? Or "ShouldProcessRuntimeParam"?

If the answer isn't clear, is this method perhaps doing too much? Could this be further broken down into different methods that make more sense?

This is also an async method and should end in "Async"